demo/example-config.yaml

```diff
@@ -1,4 +1,6 @@
 
+conversion_rules_path: demo/conversion_rules  # the directory for the conversion rules
```

Comment:
Seems like this might be better under the blueflood config section since it's only used there?